ocw_data_parser/utils.py

```diff
@@ -280,3 +286,43 @@ def parse_all(  # pylint: disable=too-many-arguments, too-many-locals
             parser.export_parsed_json(
                 s3_links=s3_links, upload_parsed_json=perform_upload
             )
+
+
+def convert_to_vtt(loaded_json):
+    """
+    Convert a json file with srt captions data to vtt format
+
+    Args:
+        loaded_json (dict): The content of JSON file
+
+    Returns:
+        dict: copy of the json file with the _datafield_file converted to vtt format
+    """
+    if loaded_json["id"].endswith(".vtt"):
+        return None
+    new_json = dict(loaded_json)
+
+    new_json["id"] = re.sub(r".srt$", ".vtt", loaded_json["id"])
+    new_json["technical_location"] = loaded_json["technical_location"].replace(
```

Comment:
Can we do the same replacement as in the previous line? It might be useful to make a helper function for that